src/cdk/package.json

```diff
@@ -53,7 +53,7 @@
     "tslib": "0.0.0-TSLIB"
   },
   "optionalDependencies": {
-    "parse5": "^5.0.0"
+    "parse5": "^7.1.2"
```

Comment:
Open question: do we want to bump this one to a release? It looks like the 7.0.0 release of parse5 was a fairly large one.